dev-tools/notice/overrides.json

```diff
@@ -1,4 +1,5 @@
 {"name": "github.com/elastic/beats/v7", "licenceType": "Elastic"}
+{"name": "github.com/AndersonQ/elastic-agent-client/v7", "licenceType": "Elastic"}
```

Comment:
It'll be removed once https://github.com/elastic/elastic-agent-client/pull/93 is merged